src/block/posts/util.js

```diff
@@ -199,7 +199,22 @@ export const generateRenderPostItem = attributes => {
 
 		// Trim the excerpt.
 		let excerptString = postExcerptStackable.split( ' ' )
-		if ( excerptString.length > ( excerptLength || 55 ) ) {
+
+		//Checks if there are CJK characters present
+		const regexCJK = /[\u3040-\u30ff\u3400-\u4dbf\u4e00-\u9fff\uf900-\ufaff\uff66-\uff9f]/
+		if ( postExcerptStackable.match( regexCJK ) ) {
+			if ( postExcerptStackable.length > excerptLength || 55 ) {
+				if ( excerptLength === 1 ) {
+					excerptString = postExcerptStackable.substring( 3, 4 ) + '...'
+				} else {
+					excerptString = postExcerptStackable.substring( 3, excerptLength === '' ? 55 : excerptLength + 3 ) + '...'
+				}
+				// Outerconditional is always true even though postExcerptStackable.length < 55.
+				if ( postExcerptStackable.length < 55 && excerptLength === '' ) {
+					excerptString = postExcerptStackable
+				}
```

Comment:
1. Why is there a different line for when the `excerptLength` is `1`, there should be no reason to split this into an `if`-`else`

2. In this `postExcerptStackable.substring( 3, 4 )`, why is the string trimming process started on index `3`? Shouldn't it start with `0`?

3. What is the last `if` statement for? Looks unnecessary.

Won't this do? Or is there an edge case where this doesn't work?
```suggestion
				excerptString = postExcerptStackable.substring( 0, excerptLength || 55 ) + '...'
```